tests/proxyname_test.go

```diff
@@ -78,55 +82,70 @@ func TestProxyNameLabel(t *testing.T) { // nolint
 			// label the disabled service
 			mustOrFatal(disabledService.SetLabel(labelKey, labelValue), t)
 			return ctx
-		}).
-		Teardown(func(context.Context, *testing.T, *envconf.Config) context.Context {
-			services := []*kubernetes.Service{disabledService.(*kubernetes.Service), toggledService.(*kubernetes.Service)}
-			tools.ResetTestBoard(t, services, model)
-			return ctx
-		}).
-		Assess("should implement service.kubernetes.io/service-proxy-name", func(_ context.Context, t *testing.T, cfg *envconf.Config) context.Context {
-			ma.Logger.Info("verify the toggledServices are up.")
+		}
+	}
+
+	teardown := func(context.Context, *testing.T, *envconf.Config) context.Context {
```

Comment:
nice, we could move this to a shared function at some point, we would need to embed the service values in the Context maybe?

Reply:
yea we can do that if more and more cases have the same teardown logic.